src/components/New Calendar/DatePicker.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState } from 'react';
+import { View, StyleSheet, TouchableOpacity, Dimensions } from 'react-native';
+import { Text, Icon } from '@ui-kitten/components';
+import { Calendar } from './Calendar';
+import { calendarCardDimensions } from '../../shared/constants/';
+
+//do we want this to be swipeable too?
```

Comment:
right now u select the month by hitting left or right arrows next to the month's name

Picker renders the calendar component and carries no state

Reply:
hm i like that idea, i'll run it by michelle and see what she thinks